packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -79,7 +79,7 @@ export const publishInterchainAssetFromBank = async (
     },
   };
   const { creatorFacet: mintP, publicFacet: issuerP } = E.get(
-    E(zoe).startInstance(mintHolder, {}, terms),
+    E(zoe).startInstance(mintHolder, {}, terms, undefined, keyword),
```

Comment:
Using just the keyword here relies somewhat on knowing that the bundle is labelled; we get `v18:zcf-mintHolder-IbcATOM`.

Elsewhere I used `${keyword}-mint` resulting in `v14:zcf-mintHolder-BLD-mint`. I'm inclined to change the BLD label.